WebRoot/WEB-INF/tags/button.tag

```diff
@@ -29,11 +29,11 @@
 <%@ taglib prefix="fmt" uri="com.zimbra.i18n" %>
 <%@ taglib prefix="app" uri="com.zimbra.htmlclient" %>
 <%@ taglib prefix="zm" uri="com.zimbra.zm" %>
-<c:if test="${not empty text}"><fmt:message key="${text}" var="text"/></c:if>
-<c:if test="${not empty tooltip}"><fmt:message key="${tooltip}" var="tooltip"/></c:if>
+<c:if test="${not empty text}"><fmt:message key="${fn:escapeXml(text)}" var="text"/></c:if>
```

Comment:
Ticket only mentions about clazz parameter which can be exploited, can you add other endpoints which were exploitable and fixed by these changes?

Reply:
Removed other variables that are not useful or out of scope.